.github/workflows/test-lib.yaml

```diff
@@ -34,3 +34,11 @@ jobs:
       - name: Run basic C++ example
         working-directory: bindings/prql-lib/examples/minimal-cpp
         run: make run
+      - name: 🔧 Setup Zig
```

Comment:
Let's do a separate job within this workflow though, so it's clearly delineated, particularly when there's a failure. You can copy & paste most things up to this line.

Reply:
Okay, I've done that now.